### DISCUSSION TITLE
New daikin MODBUS communicator EKRHH for Althermas

### DISCUSSION THREAD

**Discussion by TRIROG** [raomin/ESPAltherma#407] [Ideas]
This could be maybe useful:
https://www.daikin.eu/en_us/products/product.html/EKRHH.html#Installer-reference-guide-documents-b7f056ea11

It also contains a list of modbus registers.

**Comment by gerione** [comment]
Will get mine next week. If you need any information, just let me know.

**Reply by Buppi77** [reply]
Hi gerione, did you connected the MODBUS communicator to the Altherma? In case, can you explain me the wiring? The X7A, with connector S21, where it has to connected in the internal unit? And the X6A??
thank you

**Reply by gerione** [reply]
The X6A goes to your p1p2 connector. This is where your room controller (madoka) is connected to. The X7A is used for the S21 protocol. I think this is only used on the air conditioner units. 
I used a simple two wire cable.

**Reply by Buppi77** [reply]
Thank you. And you connected the modbus to what? PLC? Home assistant? Through rs485 cable or ethernet? Last questions, I promise

**Reply by gerione** [reply]
I connected it via modbus TCP (Ethernet) to home assistant. I am currently writing an integration(https://github.com/gerione/daikin-ha-ekrhh-modbus). But it's going slower than expected. And keep the questions coming if you have some.

**Reply by mhelle74** [reply]
Hi gerione, I have recently started using an Altherma 3 with Home Hub EKRHH and would like to try your custom integration. Can you recommend this and how do I integrate it into HA?
